slo-monitor/src/main/slo-monitor.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"net/http"
+	"net/url"
+
+	"k8s.io/contrib/cluster-autoscaler/config"
+	"k8s.io/perf-tests/slo-monitor/src/monitors"
+
+	clientset "k8s.io/client-go/kubernetes"
```

Comment:
Reshuffled. I try to keep the format:
go packages
"local" packages
various packages from kubernetes org, sorted alphabetically on repos, repos separated by single empty line
other packages